src/commands/color.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <all.hpp>
+
+class ColorCommand : public liteshell::BaseCommand
+{
+public:
+    ColorCommand()
+        : liteshell::BaseCommand(
+              "color",
+              "Change the text color in the shell",
+              "Specify a color in hex format to change the text color",
+              liteshell::CommandConstraint("color", "The color to set", true, false)) {}
+
+    DWORD run(const liteshell::Context &context)
+    {
+        if (context.values.find("color") == context.values.end())
+        {
+            std::cerr << "Error: No color specified" << std::endl;
+            return 1;
+        }
+
+        std::string color = context.values.at("color").front();
+        if (!utils::setColor(color))
+        {
+            std::cerr << "Error: Unable to set color \"" << color << "\"" << std::endl;
+            return 1;
+        }
```

Comment:
I recommend throwing `std::runtime_error` or `std::invalid_argument` instead. See other commands for details.